docs/scanners.md

```diff
@@ -171,3 +171,30 @@ class MyKeyboard(KMKKeyboard):
         # etc...
     ]
 ```
+#### Multiple Scanners coord_mapping and keymap changes
+To add more scanners you need to add onto your "coord_mapping" Example below.
+
+Coord_mapping before with just "MatrixScanner" on a 58 key split keyboard:
+```python
+coord_mapping = [
+     0,  1,  2,  3,  4,  5,         35, 34, 33, 32, 31, 30,
+     6,  7,  8,  9, 10, 11,         41, 40, 39, 38, 37, 36,
+    12, 13, 14, 15, 16, 17,         47, 46, 45, 44, 43, 42,
+    18, 19, 20, 21, 22, 23, 29, 59, 53, 52, 51, 50, 49, 48,
+            25, 26, 27, 28,         58, 57, 56, 55, 
+    ]
+```
+
+Coord_mapping after using "RotaryioEncoder" and "MatrixScanner" on the same 58 key split keyboard with an encoder on each half:
```

Comment:
```suggestion
`coord_mapping` using `MatrixScanner` and `RotaryioEncoder` on the same 58 key split keyboard with an encoder on each half:
```

I took the liberty to remove the "before/after" adjectives, because it's not necessarily a temporal relationship, rather a with-or-without thing.
Also, just to be on the safe side, I'd put the scanners in the order they are in the matrix list (the order is important).